docs/en/setup/backend/backend-k8s-monitoring-cilium.md

```diff
@@ -0,0 +1,107 @@
+        sslPrivateKeyFile: ${SW_CILIUM_FETCHER_PRIVATE_KEY_FILE_PATH:}
+        sslCertChainFile: ${SW_CILIUM_FETCHER_CERT_CHAIN_FILE_PATH:}
+        sslCaFile: ${SW_CILIUM_FETCHER_CA_FILE_PATH:}
+        convertClientAsServerTraffic: ${SW_CILIUM_FETCHER_CONVERT_CLIENT_AS_SERVER_TRAFFIC:true}
+```
+3. If enabled the [TLS certificate within the Hubble](https://docs.cilium.io/en/stable/gettingstarted/hubble-configuration/#tls-certificates), please update these few configuration.
+   1. `peerPort`: usually should be updated to the `443`.
+   2. `sslConnection`: should be set to `true`.
+   3. `sslPrivateKeyFile`: the path of the private key file.
+   4. `sslCertChainFile`: the path of the certificate chain file.
+   5. `sslCaFile`: the path of the CA file.
+
+## Generated Entities
+
+SkyWalking fetch the flow from Cilium, analyzes the source and destination endpoint to parse out the following corresponding entities:
+1. Service
+2. Service Instance
+3. Service Endpoint
+4. Service Relation
+5. Service Instance Relation
+6. Service Endpoint Relation
+
+## Generate Metrics
+
+For each of the above-mentioned entities, metrics such as L4 and L7 protocols can be analyzed.
+
+### L4 Metrics
+
+Record the relevant metrics for every service read/write packages with other services.
+
+| Name                      | Unit          | Description                                                               |
+|---------------------------|---------------|---------------------------------------------------------------------------|
+| Read Package CPM          | Count         | Total Read Package from other Service counts per minutes.                 |
+| Write Package CPM         | Count         | Total Write Package from other Service counts per minutes.                | 
+| Drop Package CPM          | Count         | Total Drop Package from other Service counts per minutes.                 |
+| Drop Package Reason Count | Labeled Count | Total Read Package reason(labeled) from other Service counts per minutes. | 
+
+### Protocol
+
+Based on each transfer data analysis, extract the information of the 7-layer network protocol.
+
+NOTE: By default, Cilium only reports L4 metrics. If you need L7 metrics, 
+they must be explicitly specified in each service's CiliumNetworkPolicy. For details please [refer to this document](https://docs.cilium.io/en/latest/security/).
+
+#### HTTP
+
+| Name                 | Unit        | Description                                             |
+|----------------------|-------------|---------------------------------------------------------|
+| Call CPM             | Count       | HTTP Request calls per minutes.                         |
+| Duration             | Nanoseconds | Total HTTP Response use duration.                       |
+| Success CPM          | Count       | Total HTTP Response success(status < 500) count.        |
+| Status 1/2/3/4/5xx   | Count       | HTTP Response status code group by 1xx/2xx/3xx/4xx/5xx. |
```

Comment:
Is 2xx in the same graph? 2xx will be super high.

Reply:
Yes, They are in the same graph.

Reply:
I think we should list all traffic in one graph and not-2xx in one graph. This should be more clear. Traffic and Anormal Traffic

Reply:
like this?
![image](https://github.com/apache/skywalking/assets/3417650/407ff433-7342-49d1-bd8a-95a42be7ba55)